src/ctelnet.h

```diff
@@ -75,7 +75,6 @@ const char MXP = 91; // MXP
 const char MSDP = 69; // MSDP, documented at http://tintin.sourceforge.net/msdp/
 
 const char OPT_ECHO = 1;
-const char OPT_SUPPRESS_GA = 3;
```

Comment:
For naming consistency why are the ones above not `OPT_GMCP`, `OPT_MXP` & `OPT_MSDP`? :confused: